Makefile

```diff
@@ -1,16 +1,23 @@
-CC = gcc
+CXX = g++
+CC = g++
 
+#purely for convinience, this way main source file can stay consistent with c/c++ ideology while letting the app have it's name
+APP_NAME = ccalc

```

Comment:
CPP calc? as its in C++ now

Reply:
and is this variable used while compiling with make?

Reply:
mainly as it doesnt seem to be used, and I am not fully sure if the executable is called this using make

Reply:
2fast4you uwu
already changed it because i forgot how makefile works for a minute uwu